### PR TITLE
Add join table for visits

### PR DESCRIPTION
**Pull request by AlaaAlsalem**
Here are the things I added to this repo
> Add query to the schema to create tables for many to many relational databases
Add a query to insert values to the visits table
Add queries to display data from the database 
Here you can find [screenshots ](https://github.com/AlaaAlsalem/Vet-clinic/tree/add_join_table_for_visits/Screenshots%20for%20select%20query)for the  result of the queries for displaying the data fromthe data base

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -94,4 +94,80 @@ SELECT full_name, COUNT(name) FROM owners O
...
+    ORDER BY COUNT(date) DESC
+    LIMIT 1;
+
+SELECT A.name FROM vets VE
+    LEFT JOIN visits VI
+    ON VE.id = VI.vet_id
+    LEFT JOIN animals A
+    ON VI.animal_id = A.id
+    WHERE VE.name = 'Maisy Smith'
+    ORDER BY VI.date DESC
```
Comment (keneogu) [inline queries.sql]: The ORDER BY VI.date should be in Ascending not descending.

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @AlaaAlsalem,

Good job so far!

- All linters are passing
- You are following GitHub flow
- Good commit history

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] The project requires you to add the screenshots on the pull request description, not a link to the GitHub folder containing the pictures.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (AlaaAlsalem) [reply queries.sql]: Thanks for your feedback, just solved

**Review by andersonlebon — APPROVED** [review]
## Hi @AlaaAlsalem 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**